easybuild/easyconfigs/v/vsearch/vsearch-2.4.3-foss-2016b.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'vsearch'
+version = '2.4.3'
+
+homepage = 'https://github.com/torognes/vsearch'
+description = """We have implemented a tool called VSEARCH which supports de novo and reference based
+ chimera detection, clustering, full-length and prefix dereplication, rereplication, reverse
+ complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching,
+ shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering, conversion and
+ merging of paired-end reads."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/torognes/vsearch/archive/']
+
```

Comment:
@verdurin Let's include a `SHA256` checksum?

Now's the perfect time to give `--inject-checksums` a try and provide feedback; you'll need to grab my branch at https://github.com/easybuilders/easybuild-framework/pull/2286 to try it.

Reply:
@verdurin ping?